input/postgres/server_stats.go

```diff
@@ -21,12 +22,76 @@ SELECT
 FROM pg_catalog.pg_control_checkpoint()
 `
 
+const xminHorizonSQL string = `
+SELECT
+COALESCE((
+	SELECT
+		CASE WHEN COALESCE(age(backend_xid), 0) > COALESCE(age(backend_xmin), 0)
+			THEN backend_xid
+			ELSE backend_xmin
+		END
+	FROM pg_stat_activity
+	WHERE backend_xmin IS NOT NULL OR backend_xid IS NOT NULL
+	ORDER BY greatest(age(backend_xmin), age(backend_xid)) DESC
+	LIMIT 1
+), '0'::xid) as backend,
+COALESCE((
+	SELECT
+		xmin
+	FROM pg_replication_slots
+	WHERE xmin IS NOT NULL
+	ORDER BY age(xmin) DESC
+	LIMIT 1
+), '0'::xid) as replication_slot_xmin,
+COALESCE((
+	SELECT
+		catalog_xmin
+	FROM pg_replication_slots
+	WHERE xmin IS NOT NULL
```

Comment:
Should this be `WHERE catalog_xmin IS NOT NULL`? (not sure if there are cases where one is set and not the other?)